docs/contributing/commit-signing.mdx

```diff
@@ -21,34 +21,38 @@ their commits.
 
 ## Setting up commit signing
 
-Github supports commit signing with GPG, SSH and S/MIME. If you're unsure what to use, we recommend
-GPG.
+Github supports commit signing with GPG, SSH and S/MIME.
 
-1.  Install GnuPG:
+:::tip
 
-  <Tabs groupId="os">
+If you're unsure what to use, we recommend you create a commit signing key using SSH. Remember to
+protect the key with a [strong passphrase or password][password-generation].
 
-    <TabItem value="mac" label="macOS">
+```bash
+ssh-keygen -f ~/.ssh/bw-signing -C "$(git config --get user.email)" -t ed25519
```

Comment:
❓ Are we implying anything with the unique name here? Versus just using the default and this be one signing key to rule them all.

Reply:
Your auth key to sign in to many computers in an interactive online orocess. Your signing key, meanwhile, only signs github commits, which is an offline process. Compartmentalizing the use-cases is best practice.

This is also a prelude to recommending keys loaded from Bitwarden, once we've released SSH support. In such a case, we'd need to identify the vault item containing the key.

Reply:
[72d55e5](https://github.com/bitwarden/contributing-docs/pull/507/commits/72d55e51863d071820cc5c6007b9077600087a11) extracts the path into a variable that's shared with the git configuration sample. This should better communicate that the the filename is arbitrary. 😄